sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/TpchTest.java

```diff
@@ -0,0 +1,853 @@
+            "                           O_SHIPPRIORITY   INTEGER NOT NULL,\n" +
+            "                           O_COMMENT        VARCHAR(79) NOT NULL);\n" +
+            "\n" +
+            "CREATE TABLE LINEITEM ( L_ORDERKEY    INTEGER NOT NULL,\n" +
+            "                             L_PARTKEY     INTEGER NOT NULL,\n" +
+            "                             L_SUPPKEY     INTEGER NOT NULL,\n" +
+            "                             L_LINENUMBER  INTEGER NOT NULL,\n" +
+            "                             L_QUANTITY    DECIMAL(15,2) NOT NULL,\n" +
+            "                             L_EXTENDEDPRICE  DECIMAL(15,2) NOT NULL,\n" +
+            "                             L_DISCOUNT    DECIMAL(15,2) NOT NULL,\n" +
+            "                             L_TAX         DECIMAL(15,2) NOT NULL,\n" +
+            "                             L_RETURNFLAG  CHAR(1) NOT NULL,\n" +
+            "                             L_LINESTATUS  CHAR(1) NOT NULL,\n" +
+            "                             L_SHIPDATE    DATE NOT NULL,\n" +
+            "                             L_COMMITDATE  DATE NOT NULL,\n" +
+            "                             L_RECEIPTDATE DATE NOT NULL,\n" +
+            "                             L_SHIPINSTRUCT CHAR(25) NOT NULL,\n" +
+            "                             L_SHIPMODE     CHAR(10) NOT NULL,\n" +
+            "                             L_COMMENT      VARCHAR(44) NOT NULL);\n" +
+            "\n" +
+            "// query source:\n" +
+            "// https://github.com/2ndQuadrant/pg-tpch/blob/master/queries\n" +
+            "\n" +
+            "// Q1\n" +
+            "create view q1 (\n" +
+            "    l_returnflag,\n" +
+            "    l_linestatus,\n" +
+            "    sum_qty,\n" +
+            "    sum_base_price,\n" +
+            "    sum_disc_price,\n" +
+            "    sum_charge,\n" +
+            "    avg_qty,\n" +
+            "    avg_price,\n" +
+            "    avg_disc,\n" +
+            "    count_order\n" +
+            ") as\n" +
+            "select\n" +
+            "    l_returnflag,\n" +
+            "    l_linestatus,\n" +
+            "    sum(l_quantity) as sum_qty,\n" +
+            "    sum(l_extendedprice) as sum_base_price,\n" +
+            "    sum(l_extendedprice * (1 - l_discount)) as sum_disc_price,\n" +
+            "    sum(l_extendedprice * (1 - l_discount) * (1 + l_tax)) as sum_charge,\n" +
+            "    avg(l_quantity) as avg_qty,\n" +
+            "    avg(l_extendedprice) as avg_price,\n" +
+            "    avg(l_discount) as avg_disc,\n" +
+            "    count(*) as count_order\n" +
+            "from\n" +
+            "    lineitem\n" +
+            "where\n" +
+            "    l_shipdate <= date '1998-12-01' - interval '71' DAY\n" +
+            "group by\n" +
+            "    l_returnflag,\n" +
+            "    l_linestatus\n" +
+            "order by\n" +
```

Comment:
Order By is not incremental, so you can remove it. Or have a copy of the view without it that we'll run.